Jenkinsfile

```diff
@@ -49,3 +55,13 @@ def runPublishingApiPactTests(govuk) {
     }
   }
 }
+def runCollectionsOrganisationApiPactTests(govuk){
```

Comment:
I think this is for a later PR, but this and `runPublishingApiPactTests` are very similar so I wonder if there's some refactoring we could do here. I suspect we'll learn this as we add pact tests to more of our apps though, most likely we'll end up putting something in [govuk-jenkinslib](https://github.com/alphagov/govuk-jenkinslib).